cassandra-release/cassandra-check-release.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash
+
+# Parameters
+# $1 release
+# $2 maven artefacts url (as specified in the vote email)
+#
+# Example use: `./cassandra-check-release.sh 4.0-beta3 https://repository.apache.org/content/repositories/orgapachecassandra-1224/org/apache/cassandra/cassandra-all/4.0-beta3/`
```

Comment:
From the security point of view, I think it is better to pass a staging number only to avoid loading artifacts from untrusted sources e.g. in this case `1224`.

Reply:
done.